src/io/openshift/plugins/analytics.groovy

```diff
@@ -5,6 +5,37 @@ import io.openshift.Events
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: 'node'
```

Comment:
should not default to "node" rather it is better to print a message here and skip analytics

Reply:
+1 unless we can confirm an image that is an appropriate default